specification/compute/resource-manager/Microsoft.Compute/RecommenderRP/models.tsp

```diff
@@ -0,0 +1,119 @@
+import "@typespec/rest";
+import "@typespec/http";
+import "@azure-tools/typespec-azure-resource-manager";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+using Azure.ResourceManager;
+using Azure.ResourceManager.Foundations;
+
+namespace Microsoft.Compute;
+
+/**
+ * Contains additional properties of a diagnostic
+ */
+#suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-provisioning-state" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+model DiagnosticProperties {
+  /**
+   * Describes what are the supported resource types for a diagnostic.
+   */
+  supportedResourceTypes?: string[];
+}
+
+/**
+ * SpotPlacementScores API Input.
+ */
+model SpotPlacementScoresInput {
+  /**
+   * The desired regions
+   */
+  desiredLocations?: string[];
+
+  /**
+   * The desired virtual machine SKU sizes.
+   */
+  #suppress "@azure-tools/typespec-azure-resource-manager/missing-x-ms-identifiers" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
```

Comment:
What does the linter error have to do with this property?

Reply:
Updated justification message